app/sequencescape_excel/sequencescape_excel/column.rb

```diff
@@ -23,7 +23,8 @@ class Column
                      defaults: {
                        number: 0,
                        type: :string,
-                       conditional_formattings: {},
+                       conditional_formattings: {
+                       },
```

Comment:
This feels like a weird change for prettier to want.  Does it still want to make this change if you separate the curly braces with a space or two?  Sometimes it's OK so long as you use `{  }` rather than two curly braces directly next to each other.  Alternatively, you can replace `{}` with `Hash.new`